docs/vision-usage.rst

```diff
@@ -31,8 +31,38 @@ or pass in ``credentials`` and ``project`` explicitly.
      >>> client = vision.Client(project='my-project', credentials=creds)
 
 
+Creating an :class:`~google.cloud.vision.image.Image` object.
```

Comment:
1. Does a header need to end in a period?
1. Isn't ending with `Image` clear? (i.e. don't say "object")